newspack-sacha/inc/child-typography.php

```diff
@@ -42,6 +42,7 @@ function newspack_sacha_custom_typography_css() {
 	if ( true === get_theme_mod( 'accent_allcaps', true ) ) {
 		$css_blocks        .= '
 			.accent-header,
+			.widgettitle,
```

Comment:
This should just be applied to when the `.widgettitle` class appears in the sidebar:

```suggestion
			#secondary .widgettitle,
```